File: history_today-master/app/src/main/java/com/animee/todayhistory/ui/HistoryDescContact.java
package com.animee.todayhistory.ui;

import com.animee.todayhistory.bean.HistoryDescBean;

public interface HistoryDescContact {
    interface View {
        void showResult(HistoryDescBean bean);
    }
    interface Presenter{
        void loadGet(String his);
    }
}

File: history_today-master/app/src/main/java/com/animee/todayhistory/ui/HistoryDescContact.kt
package com.animee.todayhistory.ui

import com.animee.todayhistory.bean.HistoryDescBean

interface HistoryDescContact {
    interface View {
        fun showResult(bean: HistoryDescBean)
    }

    interface Presenter {
        fun loadGet(his: String)
    }
}
